### ISSUE TITLE
Whitespace

### ISSUE THREAD

**Issue by imackintosh** [richorama/IronBlock#24]
Hi
I'm using the Blockly ListsSplit block to do a Join using a space as a delimiter. However, the space representing the delimiter is getting removed.
In my XML one of my fields comes through from blockly like: `<field name="TEXT"> </field>`
Note the space as the field value is still present. 
The problem appears to be with `xdoc.LoadXml(xml)` Because `xdoc.PreserveWhitespace = false` which then automatically removes any whitespace.
I was wondering if your Parse() method could be changed in Parser.cs to be:

```
        public Workspace Parse(string xml, bool preserveWhitespace = false)
        {
            var xdoc = new XmlDocument();
            xdoc.PreserveWhitespace = preserveWhitespace;
            xdoc.LoadXml(xml);
```
Thanks
Ivan

**Comment by richorama** [comment]
nice find! would you like to create a PR?

**Comment by richorama** [comment]
fixed in `1.0.7`
